pkg/sql/crdb_internal.go

```diff
@@ -200,6 +201,26 @@ CREATE TABLE crdb_internal.node_runtime_info (
 	},
 }
 
+var crdbInternalDatabasesTable = virtualSchemaTable{
+	comment: `databases accessible by the current user (KV scan)`,
+	schema: `
+CREATE TABLE crdb_internal.databases (
+	oid OID NOT NULL,
```

Comment:
Hmm, including this doesn't really fit - none of the other crdb internal tables have an oid (which is a pg_catalog concept of course) so I don't think this one should either. Looks like you need this for the comments - I think you should change the query to join through `pg_catalog.database` instead.

Reply:
I know, it's super awkward. I can't join against pg_catalog here, because show databases is supposed to work without a database context, but we can't access any of the pg_catalog tables without a database context.

Reply:
Hmm, but doesn't the `WITH COMMENT` thing need to go through `pg_catalog` right now?

Oh, I have a better idea - can we have the query just look directly from `system.comments` instead? Should be possible.

Reply:
Now this can go away I think:)

Reply:
Yeah, I'm just making sure that tests pass first